plugin-debezium/src/main/java/io/kestra/plugin/debezium/models/Message.java

```diff
@@ -58,5 +58,10 @@ public static class Source extends Message {
         public void setTsMs(Long value) {
             this.timestamp = Instant.ofEpochMilli(value);
         }
+
+        @JsonSetter("collection")
+        public void setCollection(String value) {
+            this.table = value;
+        }
```

Comment:
Instead of storing the collection inside the table, I'd rather add a collection attribute to the class so the Message object is close to the Debezium message.

Reply:
field table write unique name for file that stores data. For example if in database kestra in collection types will be changes, then if we not set collection name in field table file will have name: "kestra.null". I didn't want to rewrite much in basic classes that was before me (in plugin-debezium)

Reply:
OK then